.circleci/verdaccio.yml

```diff
@@ -24,6 +24,10 @@ packages:
   '@react-native/*':
     access: $all
     publish: $all
+    proxy: npmjs
```

Comment:
This allows us to proxy `@react-native/` packages from npm so we don't have to build them locally.

However, we want all `@callstack/` packages to come only from private registry